c2corg_ui/templates/route/view.html

```diff
@@ -1,8 +1,10 @@
 <%!
 from c2corg_ui.templates.utils import get_lang_lists
 %>\
+<%namespace file="../helpers/view.html" import="show_attr, show_short_attr, show_missing_langs_links, show_other_langs_links"/>
+
```

Comment:
It would make more sense to move this ``<%namespace`` after the ``<%inherit`` near the other ``<%namespace``.